src/lib/Session.svelte

```diff
@@ -127,10 +129,19 @@
 
   onMount(async () => {
     // The page hash sets the end-to-end encryption key.
-    const key = window.location.hash?.slice(1) ?? "";
+    const key = window.location.hash?.slice(1).split(",")[0] ?? "";
+    const writePassword = window.location.hash?.slice(1).split(",")[1] ?? null;
+
     encrypt = await Encrypt.new(key);
     const encryptedZeros = await encrypt.zeros();
 
+    write_password_encrypt = writePassword
```

Comment:
You can just save this as a temporary variable, no need to put it as a prop at the top level of the svelte component!

Also `writePasswordEncrypt` naming